armadillo_cli_add.py

```diff
@@ -0,0 +1,69 @@
+import typer
+import utils
+
+app              = typer.Typer()
+SECTION_NAME     = "policy"
+
+# Generate new section name
+def get_next_section_name(config):
+    new_section = ""
```

Comment:
```suggestion
    new_section = SECTION_NAME
```

And remove it from the following lines, this way you avoid assigning this var twice